apps/ex_wire/lib/ex_wire/config.ex

```diff
@@ -128,8 +146,7 @@ defmodule ExWire.Config do
 
   @spec client_id() :: String.t()
   def client_id() do
-    version = Mix.Project.config()[:version]
-    "mana/#{version}"
+    "mana/#{get_env([], :mana_version)}"
```

Comment:
is having a slash in the name ok? I wonder if it would interfere with directory path

Reply:
Believe so, Parity is: `client_id: "Parity-Ethereum/v2.1.3-beta-18ddd7c-20181015/x86_64-macos/rustc1.29.0"`